OP.md

```diff
@@ -0,0 +1,120 @@
+# Running an OP Mainnet or Testnet node
+
+## Non-docker configuration
+
+Here are instructions if you want to run boba erigon version as the replica node for OP Mainnet or Testnet via binary files.
+
+### Build erigon
+
+1. Clone [boba erigon](https://github.com/bobanetwork/v3-erigon) repo
+
+	```bash
+	cd ~
+	git clone https://github.com/bobanetwork/v3-erigon.git
+	```
+
+2. Build [boba erigon](https://github.com/bobanetwork/v3-erigon)
+
+	```bash
+	cd v3-erigon
+	make erigon
+	```
+
+### Build op-node
+
+1. Clone [boba anchorage](https://github.com/bobanetwork/v3-anchorage) repo
+
+   ```bash
+   cd ~
+   git clone https://github.com/bobanetwork/v3-anchorage.git
+   ```
+
+2. Build [op-node](https://github.com/bobanetwork/v3-anchorage/tree/boba-develop/op-node)
+
+	```bash
+	cd v3-anchorage/op-node
+	make
+	```
+
+### Get the data dir
+
+The next step is to download the initial data for `op-erigon`. Thanks for the contribution from [Test in Prod](https://www.testinprod.io).
+
+1. Download the correct data directory snapshot.
+   * [OP Mainnet](https://op-erigon-backup.mainnet.testinprod.io)
+   * [OP Goerli](https://op-erigon-backup.goerli.testinprod.io)
+
+2. Create the data directory in `op-erigon` and fill it.
+
+   ```bash
+   mkdir op-erigon
+   cd ./op-erigon
+   mkdir erigon
+   cd ./erigon
+   tar xvf ~/[DIR]/op-erigon-goerli.tar
+   ```
+
+3. Create a shared secret (JWT token)
+
+   ```bash
+   cd op-erigon
+   openssl rand -hex 32 > jwt.txt
+   ```
+
+### Scripts to start the different components
+
+`op-erigon`
+
+```bash
+cd v3-erigon
+./build/bin/erigon \
+--datadir=DATA_DIRECTORY \
+--private.api.addr=localhost:9090 \
+--http.addr=0.0.0.0 \
+--http.port=9545 \
+--http.corsdomain="*" \
+--http.vhosts="*" \
+--authrpc.addr=0.0.0.0 \ 
+--authrpc.port=8551 \
+--authrpc.vhosts="*" \
+--authrpc.jwtsecret=JWT_TOKEN_DIRECTORY \
```

Comment:
This should be called JWT_TOKEN_PATH as it needs the actual file, not its directory.